tests/integration/docarray_v2/test_streaming.py

```diff
@@ -143,17 +143,19 @@ async def test_streaming_delay(protocol, include_gateway):
     ):
         client = Client(port=port, protocol=protocol, asyncio=True)
         i = 0
-        start_time = time.time()
-        async for doc in client.stream_doc(
+        stream = client.stream_doc(
             on='/hello',
             inputs=MyDocument(text='hello world', number=i),
             return_type=MyDocument,
-        ):
+        )
+        start_time = None
+        async for doc in stream:
+            start_time = start_time or time.time()
```

Comment:
Looks like streaming takes longer to initialize, but has all of the data emitted at once:
```
Expected delay to be less than 0.5, got 7.867813110351562e-06 on iteration 1
Expected delay to be less than 1.0, got 0.00012993812561035156 on iteration 2
Expected delay to be less than 1.5, got 0.0001659393310546875 on iteration 3
Expected delay to be less than 2.0, got 0.00019884109497070312 on iteration 4
Expected delay to be less than 2.5, got 0.00022912025451660156 on iteration 5
``

Reply:
All these times do not include a 2.5 second initialization time.

Reply:
Just realizes the gateway seems to be breaking the streaming functionality here. Will look into why this is happening.

Reply:
Looks like the gateway might be breaking streaming.